plugins/orchestrator/package.json

```diff
@@ -72,7 +72,8 @@
     "react-use": "^17.4.0",
     "swr": "^2.0.0",
     "uuid": "^9.0.1",
-    "vscode-languageserver-types": "^3.16.0"
+    "vscode-languageserver-types": "^3.16.0",
+    "axios": "1.6.8"
```

Comment:
This is just for the tests?
Add it to devDependencies

Reply:
No it's needed in the code too see https://github.com/janus-idp/backstage-plugins/blob/9e4e66c264b1fe31d91498e124d73670c10f70c6/plugins/orchestrator/src/api/OrchestratorClient.ts#L5-L9

Reply:
right, my mistake